### ISSUE TITLE
Designated time to clean?

### ISSUE THREAD

**Issue by marykatefain** [Uptwinkles/manual#2]
I don't think there should be a designated time of day that people have to do the cleaning. They may not be available at that time or feel like it. If they do their chore in the morning or afternoon, what difference does it make?

**Comment by alexgleason** [comment]
When solicitors try to get people to vote in elections, they get them to say the time and place they're going to vote. It's a psychological trick to force them to plan it to happen, and to prevent it from not happening at all.

We could give people more freedom by using a computer to manage people. There are any number of ways. For me, this rule is a temporary compromise to keep things structured as we work towards using computers to give people more freedom. We can't afford carelessness right now.

Brainstorming software solutions:

* We could have default times for each chore, at which a bot will annoy you at on Matrix. You can reply to bot to "snooze" the time or alter it on the fly. The bot silently updates the general house chat without notifications. Or if it matters, it does send a notification. These default could be managed by the group.

* Or, chores are assigned in a dashboard which allows users to select what chores they want and pledge to do them in a certain time range. This gives us accountability, but also allows people to be flexible (they can go back and change the time). Again, a bot will notify the group of updates. Ideally, we will have some system in place to enforce fairness, and reflect that within the user interface.

Basically I want people to be accountable. I don't want to tyrannize them, but I would rather have a lot of structure with a little freedom than a little structure and a lot of freedom until we get our shit together. We can't afford to not be accountable in these beginning stages because it will bog everyone down and halt our progress. The chores must be done, and they must be done in a timely manner, every single time. Maybe being forced to not have a choice will incentivize us to actually write the software we need to get ourselves out of it.

**Comment by marykatefain** [comment]
Maybe people just have to check in by 9pm, then? I worry that not allowing people to choose their time will make them go "Oh, well I'm not free at 9pm. So guess I can't do the chore". The election people don't tell you when you have to vote, they ask you to pick a time. If they told you you should vote at 11am and then 11am came and passed, you would feel like you missed your chance to vote and not do it. So if you tell people they have to check in by 9pm to confirm their chore is complete, they will have to pick a time before then to do it. This will probably result in a lot of people doing it at 8:50pm anyways, but that's totally fine. It's more about the illusion of freedom anyways.

I like the idea of a bot reminder in the morning: 9am: "Reminder: Clean the kitchen today by 9pm" and then at 9pm: "Did you clean the kitchen? y/n?"

**Comment by alexgleason** [comment]
>The election people don't tell you when you have to vote, they ask you to pick a time.

I used this example to convey the psychological tactic, but I don't think it applies beyond that. Sure, we're beholden to a voter in the sense that they contribute to our political future, but the scale for which we rely on them is vastly different than for members of our household. One person's failure to comply at the house could break the whole system. The person who empties the dishwasher relies on the dishes being washed and loaded, and the people who wash dishes need the precedent of an empty sink to be motivated to wash their dishes.

We also have more leverage in this case to force people to comply, since they face the risk of being removed from their home if they don't comply. I kind of wish people had this level of incentive to vote. They might actually do something for a change. Obviously I don't condone this in general, but the house is a hotbed. I'm fine implementing Stalinism, giving the group consensus, and constantly maintaining the status quo (as long as the system is still working) until we implement software to incrementally transcend into anarchist glory. I don't trust people to be adults anymore unless we lay it out for them in step-by-step instructions. Myself included, honestly. I wonder if Stalin became this way after losing faith in humanity.

Basically, I agree with you, but what else can we do at this time that will actually work? Writing in your own time on the sheet won't cut it since it won't notify the house and there's no way to enforce it or make people accountable. Enforcing it would require long house meetings where everyone writes in their times, announces it, and we come to consensus that it's okay. But the house needs to know what times things are happening at, and people need to be held accountable to actually do the chores.

>"Oh, well I'm not free at 9pm. So guess I can't do the chore"

For one, we can mitigate this by making our schedule clear to newcomers. If they have scheduling problems, they should reconsider living with us, or they should schedule around doing the chores. We should accommodate people with a normal 9-5 schedule since this should cover most people. If you can't do something on time, you're responsible for asking someone else to do it for you. If that doesn't work, you're responsible for doing it at the earliest possible time or you risk failing the group.

I don't think we would lose peoples' sense of involvement or stake in the house as long as we stay non-hierarchical. But people should fully understand why things are this way in the first place. I would tolerate people acting out of accordance with the schedules if they're bringing forward a solution to implement it better.

**Comment by alexgleason** [comment]
We're following Twink now
